cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -229,6 +229,10 @@ def validate_backup_id_unsupported_parameters(self, data, **kwargs):
 class SharedStorageSchema(BaseSchema):
     """Represent the generic SharedStorage schema."""
 
+    # FIXME: pcluster2 doesn't allow "^/?NONE$" mount dir to avoid an ambiguity in cookbook.
```

Comment:
I'd add this check in the SharedDirValidator, to merge a working code.

Reply:
Done